internal/csi-addons/networkfence/fencing.go

```diff
@@ -286,6 +298,113 @@ func (nf *NetworkFence) AddClientEviction(ctx context.Context) error {
 	return nil
 }
 
+func parseBlocklistEntries(output string) []string {
+	// Parse the stdout to extract blocklist entries
+	// variable entries is a slice, contain the output of above command as separate strings
+	// it will remove any empty lines and the 1st line which is "listed x entries"
+	// example, entries[0] = "127.0.0.1:0/3710147553 2018-03-19 11:32:24.716146"
+	lines := strings.Split(output, "\n")
+	var entries []string
+	for _, line := range lines {
+		if strings.TrimSpace(line) != "" && !strings.HasPrefix(line, "listed") {
+			entries = append(entries, line)
```

Comment:
Are complete entry lines used anywhere? If not, you could call `fetchIPfromEntry()` here and add the IP-address to the `[]string` that is returned.

If that is an option, you could rename `getBlocklistEntries()` to `getBlocklistIPs()` and make the code a little simpler.

Reply:
Thanks, @nixpanic 
Updated, PTAL.